src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -194,253 +328,349 @@ const CurationPage = (props: ICurationPageProps) => {
+                                      </div>
+                                    </Collapsible>
+                                  );
+                                })
+                              );
+                            }, [])}
+                          </Collapsible>
+                        );
+                      })}
+                  </Collapsible>
+                </Col>
+              </Row>
+            ))}
           </div>
         )}
         <VusTable hugoSymbol={hugoSymbol} />
+        <Modal isOpen={showFilterModal} toggle={handleToggleFilterModal}>
+          <ModalHeader>
+            <Container>
+              <Row>
+                <Col>Filters</Col>
+              </Row>
+            </Container>
+          </ModalHeader>
+          <ModalBody>
+            <Container>
+              <h6 className="mb-2">Oncogenicity</h6>
+              <Row>
+                {oncogenicityFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setOncogenicityFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Mutation effect</h6>
+              <Row>
+                {mutationEffectFilter.map((filter, index) => (
+                  <Col className="col-6" key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setMutationEffectFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
+                    </InputGroup>
+                  </Col>
+                ))}
+              </Row>
+              <h6 className="mb-2 mt-2">Therapeutic implication level of evidence</h6>
+              <Row className="align-items-start justify-content-start">
+                {txLevelFilter.map((filter, index) => (
+                  <Col style={{ flexGrow: 0 }} key={filter.label}>
+                    <InputGroup>
+                      <Input
+                        onChange={() => handleFilterCheckboxChange(index, setTxLevelFilter)}
+                        checked={filter.selected}
+                        style={{ marginLeft: '0px' }}
+                        type="checkbox"
+                      />
+                      <Label style={{ marginLeft: '18.75px' }}>{filter.label}</Label>
```

Comment:
I haven't looked at this PR locally, so wondering what the 18.75px is used for? Is it for centering the label?

Reply:
The checkbox from reactstrap has a left margin of -18.75px for some reason. This is to account for that and make the spacing even on both sides.